modules/nw-ovn-kubernetes-features.adoc

```diff
@@ -2,14 +2,20 @@
 //
 // * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
 
-[id="nw-ovn-kubernetes-features_{context}"]
-= OVN-Kubernetes features
+[id="nw-ovn-kubernetes-purpose_{context}"]
+= OVN-Kubernetes purpose
 
-The OVN-Kubernetes network plugin implements the following features:
+The OVN-Kubnernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubnernetes network plugin:
```

Comment:
@ahardin-rh This line introduced 2 instances of a typo "Kubnernetes". Would you mind submitting a followup PR to fix it?